subprojects/dependency-management/src/testFixtures/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformTestFixture.groovy

```diff
@@ -70,14 +70,23 @@ allprojects {
     artifacts {
         implementation producer.output
     }
+
+    // implementation (as created by the automatically applied java-library plugin in some of these tests) is not resolvable
+    // so, to handle tests using buildSrc, we'll extend it with a new, resolvable configuration and resolving that one instead
+    configurations {
+        extender {
```

Comment:
IMO the name extender does not really describe what the configuration is being used for. What do you think about `resolver` here instead?

Reply:
That's better, I'll change it.